### DISCUSSION TITLE
Feature request: Party member hotkeys from UltimaHacks

### DISCUSSION THREAD

**Discussion by InterClaw** [exult/exult#387] [General]
I searched around here, but couldn't find anybody bringing this up, so here it is.

Exult is really nice, but something that I also really like from UltimaHacks is how the hotkeys for party members work.
https://github.com/JohnGlassmyer/UltimaHacks#ultima-vii-the-black-gate

Especially the ability to open the backpacks with `Shift + Number` and to put stuff in backbacks with _drag item_ + `Number`. Those are the most useful ones.

How about replicating this control scheme in Exult as an input option? Default off by all means.

**Comment by DominusExult** [comment]
Always nice to see what else John has hacked into the original :)
Adding all this might be a good task eventually. Probably a good one for someone wanting to get into working with Exult.

The numbers do already open the paperdoll of the party members but that's it. for quick drag'n'drop the status bar with the party faces is a good replacement.

**Comment by InterClaw** [comment]
> The numbers do already open the paperdoll of the party members but that's it. for quick drag'n'drop the status bar with the party faces is a good replacement.

Ah, I didn't know about that one.

**Comment by wench** [comment]
If we do decide to add modern quality of life features like this I'm thinking we should add a setting in exult.cfg that can globally disable all non cheating QOL features the original games didn't have. Some people really do not like such features. At the moment we have a settting for cheat features but not non cheat features.
